### ISSUE TITLE
Refresh on Profile page is not working 

### ISSUE THREAD

**Issue by harsh-100** [devashishsoni98/moviebazaar#1]
Changes that are done on the profile page are amazing in the movie Bazaar app 

- There is one small issue in that page
- When , you refresh on the profile page , it will give you page not found 
- so kindly look into in this issue , 

HINT : There , is something to be done in the route part or netlify route setting also 😉

**Comment by devashishsoni98** [comment]
Thank you very much for pointing out this issue. 👍
 I searched about it and found that when the page is refreshed , it sends a request to the server side but in this case, the server does not have the route defined for that path, hence it gives an error. To fix this, I have added a _redirects file which redirect all the requests to my index.html file with a status code of 200 (OK). 

The _redirects file works as a Netlify configuration file and defines redirection rules for the  site.
It should be working fine now, Kindly check 😄. 

Thank you again for pointing out this problem.  👍
